build/Dockerfile

```diff
@@ -151,6 +151,9 @@ RUN if [[ -z "${WHEEL_VERSION}" ]]; \
     fi && \
     ls /tmp/*.whl >/tmp/bdist_name
 
+# Ensures to always build mamba_ssm from source
+ENV PIP_NO_BINARY=mamba-ssm,mamba_ssm
```

Comment:
Lets make sure that selective_scan binaries also built which may get ignored.

```
FORCE_BUILD=TRUE
SKIP_CUDA_BUILD=FALSE
```
is used during `pip install .`

Reply:
<img width="536" height="133" alt="image" src="https://github.com/user-attachments/assets/592d66f0-410f-4a46-be40-682b69c999e1" />

Reply:
looks good